Makefile

```diff
@@ -1,10 +1,13 @@
 .PHONY: build
-build: ## Build the book
+## Build the book
```

Comment:
Sorry, could you put these documentation comments back, it's used by the `help` command

Reply:
NP, I suspected it was something like that, but was apparently not observant enough to see the help rule right there! I just moved them when I added a dependency because it made the line quite long so it was a pointless change (especially now that the build is inlined).